### ISSUE TITLE
Not Receiving Shares

### ISSUE THREAD

**Issue by Azzult** [kamronbek29/tiktok_views#3]
Ok it says it sent 100 shares, I made sure it was the same URL but instead I get nothing.

**Comment by mikejack99** [comment]
same in my case :/

**Comment by kamronbek29** [comment]
@Azzult @mikejack99 Hi, try to send billions of shares, sometimes tiktok not giving shares. I tried myself, even if i got 15 shares, in couple of hours I received 8000 views. @ttsavebot on tiktok, you may check

**Comment by Azzult** [comment]
How do I change the amount of shares? It just automatically closes at 1k for me

**Comment by mikejack99** [comment]
> How do I change the amount of shares? It just automatically closes at 1k for me

is the fist question of the code ( the amount of shares).

**Comment by mikejack99** [comment]
> @Azzult @mikejack99 Hi, try to send billions of shares, sometimes tiktok not giving shares. I tried myself, even if i got 15 shares, in couple of hours I received 8000 views. @ttsavebot on tiktok, you may check

I receive the shares, but only 0,02% of i send.... (i send 235000 and received 55) :/

**Comment by kamronbek29** [comment]
> > @Azzult @mikejack99 Hi, try to send billions of shares, sometimes tiktok not giving shares. I tried myself, even if i got 15 shares, in couple of hours I received 8000 views. @ttsavebot on tiktok, you may check
> 
> 
> 
> I receive the shares, but only 0,02% of i send.... (i send 235000 and received 55) :/

Yeah, its because tiktok added signature to their requests, without it the percentage of successful requests are very low. Because of it, even if you ask millions of shares, u will get only 1000

**Comment by jedison** [comment]
@kamronbek29 thanks, that is interesting. Is there a way to add signatures to requests so that shares are received?
